modules/mysql/charts.go

```diff
@@ -462,6 +462,34 @@ var charts = Charts{
 			{ID: "open_tables", Name: "tables"},
 		},
 	},
+	{
+		ID:    "process_list_fetch_duration",
+		Title: "Process List Fetch Duration",
+		Units: "milliseconds",
+		Fam:   "process list",
+		Ctx:   "mysql.process_fetch_query_duration",
+		Dims:  Dims{{ID: "process_fetch_query_duration", Name: "Query Duration"}},
+	},
+	{
+		ID:    "process_list_query_count",
+		Title: "Queries Count",
+		Units: "queries",
+		Fam:   "process list",
+		Ctx:   "mysql.process_list_query_count",
+		Type:  module.Stacked,
+		Dims: Dims{
+			{ID: "process_list_query_count_system", Name: "System"},
+			{ID: "process_list_query_user", Name: "Users"},
+		},
```

Comment:
```suggestion
		ID:    "process_list_queries_count",
		Title: "Queries Count",
		Units: "queries",
		Fam:   "process list",
		Ctx:   "mysql.process_list_queries_count",
		Type:  module.Stacked,
		Dims: Dims{
			{ID: "process_list_queries_count_system", Name: "system"},
			{ID: "process_list_queries_user", Name: "user"},
		},
```

⚠️ don't submit the changes via GitHub

- "process_list_query_count" = "process_list_queries_count"
- lowercase dimensions names

Reply:
`process_list_queries_user should` be `process_list_queries_count_user` following the pattern

Reply:
yeah, you are right, thanks!